### ISSUE TITLE
Forums graded by Scale

### ISSUE THREAD

**Issue by elinpy** [academic-moodle-cooperation/moodle-gradereport_gradedist#7]
Hi

Teachers in my college used to use scales (stars) to grade Forums.  Those items are showing in Gradebook properly.  However, I found those items missing in the grade distribution. I can't even find them in the grading column drop-down list. Is there a way to include those Forums in the grade distribution report?

**Comment by goggo24** [comment]
You are right, these columns are not shown, because the gradedistribition only supports grading columns with grade type VALUE. We have not had any requests in this direction yet. 
We will update the readme and mention this info.
